### DISCUSSION TITLE
How Can I use Rembg with Flutter/Dart?

### DISCUSSION THREAD

**Discussion by shaadart** [danielgatis/rembg#579] [Q&A]
I was thinking it would be so helpful if rembg can be used outside of python and can be used in other languages, most significantly talking about flutter. 

How can I achieve this?

**Comment by m-aqsam** [comment]
To use rembg in Flutter, create a platform channel for communication between Dart and native code. Utilize a Dart wrapper and implement platform-specific code in Android and iOS to integrate rembg for background removal.

**Dart Wrapper :**

```
import 'dart:async';
import 'package:flutter/services.dart';


class RembgFlutter {
  static const MethodChannel _channel = MethodChannel('rembg_flutter');

  static Future<String?> removeBackground(String imagePath) async {
    try {
      final String result = await _channel.invokeMethod('removeBackground', {'imagePath': imagePath});
      return result;
    } catch (e) {
      print('Error: $e');
      return null;
    }
  }
}


```
